content/en/hosting/4.x/production/google-cloud/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Production Hosting CHT 4.x - Google Cloud Platform"
+linkTitle: "Google Cloud Platform"
+weight: 10
+aliases:
+   - /hosting/4.x/production/docker/google-cloud/
+description: >
+   Production Hosting CHT 4.x - Google Cloud Platform
+---
+This for a step by step to migrate or deploy CHT 4.x instance and is the recommended solution  on Google Cloud Platform.
+
+## Prerequisites
+
+Before you can start using  **Google Cloud Platform (GCP)** , you need to meet certain requirements
+
+1. **Google Account:** a valid Google account (Gmail or Google Workspace)
+2. **Enable Billing & Free Trial ($300 Credit)**: a valid **credit card** (identity verification)
+3. **Create a GCP Project**: each resource in GCP must belong to a **project**
+
+## Template resources, changes required and some considerations
+
+Let's start writing out the detail as to what and why, and gathering those links on info so the cht-docs PR will be easier.
+For example, what is the main docker image constraint (base image selection)? Why don't we want to use google-optimized container images, or Amazon-optimized container images? Why do we need to modify u-limit (and link to the relevant couchdb performance doc)
+
+## **Recommendations and considerations**
+
+### Why not to use optimized container images and why modify Ulimit ?
+
+On a system with many databases or many views, CouchDB can very rapidly hit the Ulimit, and this is not possible with either Google-optimized container images or Amazon-optimized container images. [More details for couchDB performance](https://docs.couchdb.org/en/stable/maintenance/performance.html)
+
+This is why we use Ubuntu with containerd (ubuntu_containerd)
+
+### Cluster creation
+
+#### Zonal cluster
```

Comment:
Let's add details around the considerations and limitations for this type of decision.

As a system support administrator, the impacts of a Zonal cluster mean there is only 1 control plane server. If that control plane server goes down, I am unable to modify, or create any resources inside the Cluster. The workloads are still running and available to the end-users but the administrator should be aware of the constraint. Easing this burden requires an increase in costs (more control plane server redudancy)